src/main/java/com/team/buddyya/certification/dto/response/VerifyCodeResponse.java

```diff
@@ -0,0 +1,3 @@
+package com.team.buddyya.certification.dto.response;
+
+public record VerifyCodeResponse(String phoneNumber, String status) {}
```

Comment:
```suggestion
public record VerifyCodeResponse(
        String phoneNumber, 
        String status) {
        }
```

필드가 여러 개 있을 때 띄어 쓰는 것이 가독성이 조금 더 좋아 보여요!

Reply:
저도 동의하는 부분입니다. 수정하였습니다!